addons/project_todo/static/src/views/todo_form/todo_form_control_panel.xml

```diff
@@ -8,14 +8,49 @@
             </t>
         </xpath>
 
+        <xpath expr="//div[hasclass('o_control_panel_navigation')]/t[@t-if='state.embeddedInfos.embeddedActions?.length and env.isSmall']" position="before">
+            <t t-if="!env.isSmall">
+                <t t-slot="todo-tag-ids"/>
+                <t t-slot="todo-user-ids"/>
+                <t t-slot="todo-personal-stage"/>
+            </t>
+            <xpath expr="//div[hasclass('o_control_panel_main_buttons')]" position="move"/>
+            <t t-if="env.isSmall">
+                <t t-slot="todo-personal-stage"/>
+                <t t-slot="todo-user-ids"/>
+                <t t-slot="todo-tag-ids"/>
+            </t>
```

Comment:
We should manage with the order instead.[order](https://getbootstrap.com/docs/5.2/utilities/flex/#order)